src/js/controllers/index.js

```diff
@@ -549,9 +548,8 @@ angular.module('copayApp.controllers').controller('indexController', function($r
     c.startScan({
       includeCopayerBranches: true,
     }, function(err) {
-      if (err) {
-        if (self.walletId == walletId)
-          self.setOngoingProcess('scanning', false);
+      if (err && self.walletId == walletId) {
```

Comment:
Not sure about this change. If an error occur and the wallet is focused it should be shown.